src/pages/EditItemPage.js

```diff
@@ -211,7 +211,18 @@ const EditItemPage = () => {
                     </FormControl>
                   </Box>
 
-                  <TextField label='Preço' type='text' variant='outlined' fullWidth required sx={editItemClasses.formField} onChange={(e) => handlePrice(e)} error={priceError} value={price} />
+                  <TextField
+                    label='Preço'
+                    type='text'
+                    variant='outlined'
```

Comment:
Change these types to something like that:

```
variant={TextFieldVariant.outlined}
type={TextFieldType.text}
```
You can set default values and then you don´t have to write this all the time. It´s not a good thing to write these props as strings because one misspelling and the component will not work

Reply:
I created a object holding all strings for `EditItemPage` and replace the strings with the values. Have a look now.